project_transportonly/project.json5

```diff
@@ -0,0 +1,351 @@
+                file: '../project/dimensions/supplemental/reeds_pca.csv',
+                module: 'dsgrid.dimension.standard',
+                mapping: {
+                    description: 'Maps US Counties 2020 L48 to ReEDS Power Control Areas',
+                    file: '../project/dimension_mappings/lookup_county_to_reeds_pca.csv',
+                    mapping_type: 'many_to_one_aggregation',
+                },
+            },
+            {
+                // For regional reporting - Roughly aligned with EPA eGrid, which is 
+                // similar to EMM, e.g., compare Figure 6 in https://www.nrel.gov/docs/fy23osti/84916.pdf
+                // to https://www.eia.gov/outlooks/aeo/pdf/nerc_map.pdf
+                type: 'geography',
+                class: 'GeographyDimensionBaseModel',
+                name: 'Cambium GEA',
+                display_name: 'Cambium GEA',
+                description: 'Cambium Generation and Emission Assessment Regions, see https://www.nrel.gov/docs/fy23osti/84916.pdf',
+                file: '../project/dimensions/supplemental/cambium_gea.csv',
+                module: 'dsgrid.dimension.standard',
+                mapping: {
+                    description: 'Maps US Counties 2020 L48 to Cambium Generation and Emission Assessment (GEA) Regions',
+                    file: '../project/dimension_mappings/lookup_county_to_cambium_gea.csv',
+                    mapping_type: 'many_to_one_aggregation',
+                },
+            },
+            {
+                class: 'GeographyDimensionBaseModel',
+                description: 'EMM Region',
+                file: '../project/dimensions/supplemental/emm.csv',
+                module: 'dsgrid.dimension.standard',
+                name: 'EMM Region',
+                display_name: 'EMM Region',
+                type: 'geography',
+                mapping: {
+                    description: 'Maps EMM regions to counties',
+                    file: '../project/dimension_mappings/lookup_county_to_emm.csv',
+                    mapping_type: 'many_to_one_aggregation',
+               },
+            },
+            {
+                // For regional reporting - Census Division does not align well with
+                // EMM/eGrid/GEA regions, so only report groups of states at this much 
+                // higher level
+                type: 'geography',
+                class: 'CensusRegion',
+                name: 'Census Region',
+                display_name: 'Census Region',
+                description: 'US Census Regions',
+                file: '../project/dimensions/supplemental/census_regions.csv',
+                module: 'dsgrid.dimension.standard',
+                mapping: {
+                    description: 'Maps US Counties 2020 L48 to CensusRegion',
+                    file: '../project/dimension_mappings/lookup_county_to_censusregion.csv',
+                    mapping_type: 'many_to_one_aggregation',
+                },
```

Comment:
We skip census division level for the main project because of the buildings dataset, but that concern doesn't apply here. Thus, you could add Census Division back in if you want.